include/openssl/http.h

```diff
@@ -33,8 +33,8 @@ extern "C" {
 # define OPENSSL_HTTP_PROXY "HTTP_PROXY"
 # define OPENSSL_HTTPS_PROXY "HTTPS_PROXY"
 
-#define HTTP_DEFAULT_MAX_LINE_LENGTH (4 * 1024)
-#define HTTP_DEFAULT_MAX_RESP_LEN (100 * 1024)
+#define OSSL_HTTP_DEFAULT_MAX_LINE_LENGTH (4 * 1024)
```

Comment:
Why is one named with _LENGTH and the other with _LEN suffix? Could you unify to _LEN as the names are long enough?

Reply:
Indeed `OSSL_HTTP_DEFAULT_MAX_LINE_LEN` is better - changed.